build/BuildDefaults.props

```diff
@@ -2,11 +2,12 @@
   <PropertyGroup>
     <CLITargets Condition=" '$(CLITargets)' == '' ">Prepare;Compile;Test;Package;Publish</CLITargets>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <HasAdditionalSharedFramework Condition="'$(Rid)' != 'linux-x64'">true</HasAdditionalSharedFramework>
+    <HasAdditionalSharedFramework Condition=" '$(HasAdditionalSharedFramework)' == '' ">false</HasAdditionalSharedFramework>
```

Comment:
I thought so too, but it wasn't clear to me why we had these two properties to begin with.